### ISSUE TITLE
Unable to click tray icon with VoiceOver

### ISSUE THREAD

**Issue by oysteinmoseng** [electron/electron#21062]
### Preflight Checklist

* [ x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [ x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 7.1.1
* **Operating System:**
  * Mac OS 10.15.1
* **Last Known Working Electron version:**
  * N/A

### Expected Behavior
When clicking tray icons using the VoiceOver screen reader, the tray click event should fire.

### Actual Behavior
Clicking the tray icon does nothing, unless a context menu is set using `tray.setContextMenu`, in which case the menu opens.

### To Reproduce
Basic demo of the issue (can be reproduced in Electron Fiddle):
https://gist.github.com/oysteinmoseng/bec736b1c2ceec3fa6e5a77d46d3ac08

To reproduce with VoiceOver, start VoiceOver (cmd+F5), navigate to the tray (VO + M M), and attempt to click the icon (VO + spacebar). VoiceOver will announce that the icon was pressed, but no click event will be fired.

**Comment by MarcoZehe** [comment]
This is still an issue in Electron 11.0 on MacOS 11.1, on both an Intel and an M1 Mac. An example app where I just encountered this was Text Expander 6.8.1 built for M1. Hoping to give this issue a bump.

**Comment by raksooo** [comment]
I can confirm that this is still an issue in Electron 21.1.0 on macOS 12.6

**Comment by raksooo** [comment]
It's only been three months since I confirmed it's still an issue on 21.1.0 after initially being reported on 7.1.1. I've kept track of the changelogs and I haven't seen anything related between 21.1.0 and 21.3.4. I'll try to reproduce it if I have time but I got a lot of notifications three months ago and a lot in the past few days so I'll see how much time I have to reproduce all issues I was notified about.

**Comment by srilakshmikanthanp** [comment]
Any update on this? I am still facing this issue on electron 22.0.2

**Comment by srilakshmikanthanp** [comment]
Any update on this? I am still facing this issue on electron 22.0.2

**Comment by srilakshmikanthanp** [comment]
Any update on this ?
